spec/requests/api/v1/npq_applications_spec.rb

```diff
@@ -229,19 +234,24 @@
     end
 
     context "when participant has applied for multiple NPQs" do
-      let!(:other_npq_profile) { create(:npq_validation_data, npq_lead_provider: npq_lead_provider, user: user) }
-      let!(:other_accepted_npq_profile) { create(:npq_validation_data, npq_lead_provider: npq_lead_provider, user: user, lead_provider_approval_status: "accepted") }
+      let!(:other_npq_validation_data) { create(:npq_validation_data, npq_lead_provider: npq_lead_provider, user: user) }
+      let!(:other_accepted_npq_validation_data) { create(:npq_validation_data, npq_lead_provider: npq_lead_provider, user: user, lead_provider_approval_status: "accepted") }
+
+      before do
+        NPQ::CreateOrUpdateProfile.new(npq_validation_data: other_npq_validation_data).call
```

Comment:
I like using services to set up test data, factories are cool and all, but this is nicer imo.